build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/NiABuildType.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid
+
+/**
+ * This is shared between :app and :benchmarks module to provide configurations type safety.
+ */
+@Suppress("unused")
+enum class NiABuildType(val applicationIdSuffix: String? = null) {
```

Comment:
nit: class for supporting flavors is just `Flavor` but this one is `NiaBuildType` (**Nia** prefix). Maybe make it consistent?

Reply:
Renamed 👍🏼